src/core/operations/ChangeIPFormat.mjs

```diff
@@ -37,6 +37,28 @@ class ChangeIPFormat extends Operation {
                 "value": ["Dotted Decimal", "Decimal", "Octal", "Hex"]
             }
         ];
+        this.checks =
```

Comment:
I'd probably remove this one. I don't think this is something we want Magic suggesting.